components/odh-notebook-controller/controllers/notebook_controller_test.go

```diff
@@ -659,34 +672,56 @@ var _ = Describe("The Openshift Notebook controller", func() {
 			Expect(cli.Create(ctx, notebook)).Should(Succeed())
 			time.Sleep(interval)
 
+			By("By fetching the created Notebook for debugging")
+			actualNotebook := &nbv1.Notebook{}
+			key := types.NamespacedName{Name: Name, Namespace: Namespace}
+			Expect(cli.Get(ctx, key, actualNotebook)).Should(Succeed())
+
 			By("By checking that the webhook has injected the sidecar container")
-			Expect(*notebook).To(BeMatchingK8sResource(expectedNotebook, CompareNotebooks))
+			Expect(actualNotebook.Spec.Template.Spec.Volumes).To(ConsistOf(expectedNotebook.Spec.Template.Spec.Volumes))
+			Expect(actualNotebook.Spec.Template.Spec.Containers).To(ConsistOf(expectedNotebook.Spec.Template.Spec.Containers))
+
 		})
 
 		It("Should remove the reconciliation lock annotation", func() {
+			By("By waiting before checking the reconciliation lock annotation removal")
+			time.Sleep(2 * time.Second)
```

Comment:
This goes against this effort for test speed-up, but ok for now
* https://github.com/opendatahub-io/kubeflow/issues/380

Reply:
removed it